src/sentry/workflow_engine/models/json_config.py

```diff
@@ -8,13 +7,9 @@
 class JSONConfigBase(models.Model):
     config = models.JSONField(db_default={})
 
-    @abstractproperty
-    def CONFIG_SCHEMA(self) -> dict[str, Any]:
-        pass
-
-    def validate_config(self) -> None:
+    def validate_config(self, schema: dict[str, Any]) -> None:
         try:
-            validate(self.config, self.CONFIG_SCHEMA)
+            validate(self.config, schema)
```

Comment:
updated, instead we pass in the schema instead of having it defined on the class, and use a pre-save signal to call this function